pom.xml

```diff
@@ -92,34 +92,34 @@
     <hikaricp.version>4.0.3</hikaricp.version>
     <jansi.version>2.4.0</jansi.version>
     <jenkins-annotation-indexer.version>1.17</jenkins-annotation-indexer.version>
-    <jenkins-credentials.version>1143.vb_e8b_b_ceee347</jenkins-credentials.version>
+    <jenkins-credentials.version>1189.vf61b_a_5e2f62e</jenkins-credentials.version>
```

Comment:
Are you sure you still need to override all these plugin versions rather than just picking up compatible versions from the BOM?

Reply:
Good point, I need to remember why those overrides have been added

Reply:
It is possible they predated the plugin BOM. Normally when starting to use the BOM for the first time, you should do a sweep of all dependencies in the POM, deleting `version` from any dependency that is now managed, and even deleting transitive dependencies that are not explicitly used from the project. Also note that there is a complementary core BOM imported by default by the `plugin-pom` parent which manages dependencies of core libraries, so these should not be explicitly mentioned in the POM either.